.eslint-dictionary.json

```diff
@@ -163,4 +166,4 @@
   "xcode",
   "yaml",
   "yyyymmddhhmmss"
-]
+]
```

Comment:
nit: run `yarn mergewords` so this list will be sorted and the trailing newline should be added back